roles/glance/templates/etc/glance/policy.json

```diff
@@ -16,8 +16,13 @@
     "upload_image": "",
 
     "delete_image_location": "",
+{% if allow_image_locations|default('False')|bool %}
```

Comment:
It would probably be better to have this variable as glance.allow_image_locations IMO.